packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -75,6 +88,25 @@ abstract class GestureRecognizer extends GestureArenaMember with DiagnosticableT
   /// recognizer looks for, like 'tap' or 'horizontal drag'.
   String get debugDescription;
 
+  /// {@template flutter.gestures.recognizer.dragStartBehavior}
+  /// Configure the behavior of offsets sent to [onStart].
+  ///
+  /// If set to [DragStartBehavior.start], the [onStart] callback will be called at the time and
+  /// position when the gesture detector wins the arena. If [DragStartBehavior.down ],
+  /// [onStart] will be called when at the time and position when a down event
+  /// was first detected.
+  ///
+  /// For example:
+  /// A finger presses down at on the screen with offset (500.0, 500.0),
+  /// and then moves to position (510.0, 500.0) before winning the arena.
+  /// With [dragStartBehavior] set to [DragStartBehavior.down], the [onStart]
+  /// callback will be called at the time corresponding to the touch's position
+  /// at (500.0, 500.0). If it is instead set to [DragStartBehavior.start],
+  /// onStart will be called at the time corresponding to the touch's position
+  /// at (510.0, 500.0).
+  /// {@endtemplate}
+  DragStartBehavior dragStartBehavior;
```

Comment:
this should just be on the DragGestureRecognizer class

Reply:
Again, I don't think so, because there are some widgets that create GestureRecognizers directly so it makes more sense for GestureRecognizer to have it because DragGestureRecognizer inherits from it.

Reply:
I don't follow. This class doesn't use this field, so by definition it doesn't belong on this field.